package/samplers/cmamae/example.py

```diff
@@ -1,22 +1,24 @@
 import optuna
-from optuna.study import StudyDirection
 import optunahub
 
 
 module = optunahub.load_module("samplers/cmamae")
 CmaMaeSampler = module.CmaMaeSampler
 
 
-def objective(trial: optuna.trial.Trial) -> tuple[float, float, float]:
+def objective(trial: optuna.trial.Trial) -> float:
     """Returns an objective followed by two measures."""
     x = trial.suggest_float("x", -10, 10)
     y = trial.suggest_float("y", -10, 10)
-    return x**2 + y**2, x, y
+    trial.set_user_attr("x", x)
```

Comment:
I would set the measure names to be `m0` and `m1` rather than `x` and `y` so that it is more clear that measure names need not be the same as the parameters. For example, we could easily have:
```
trial.set_user_attr("m0", 2 * x)
trial.set_user_attr("m1", x + y)
```